src/test/resources/rca/bucketization/bucketize.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Same as below. Can we make this a string and pass to rca.conf. 

It makes the mapping better (which test is using which input)

Reply:
got it !